sub/project_settings/{{ cookiecutter.__folder_name }}/backend/pyproject.toml

```diff
@@ -0,0 +1,188 @@
+    "plone.api",
+    "plone.restapi",
+    "plone.volto",
+]
+
+[project.optional-dependencies]
+test = [
+    "plone.app.testing",
+    "plone.restapi[test]",
+    "pytest",
+    "pytest-cov",
+    "pytest-plone>=0.5.0",
+]
+
+[project.urls]
+Homepage = "https://github.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.project_slug }}"
+PyPI = "https://pypi.org/project/{{ cookiecutter.python_package_name }}"
+Source = "https://github.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.project_slug }}"
+Tracker = "https://github.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.project_slug }}/issues"
+
+{%- if cookiecutter.plone_version.startswith('6.1') is true %}
+[project.entry-points."plone.autoinclude.plugin"]
+{%- else %}
+[project.entry-points."z3c.autoinclude.plugin"]
+{%- endif %}
+target = "plone"
+
+[tool.uv]
+managed = {{ cookiecutter.__backend_managed_by_uv }}
+
+[tool.hatch.version]
+path = "src/{{ cookiecutter.__package_path }}/__init__.py"
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build]
+strict-naming = true
+
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "/.github",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/{{ cookiecutter.__package_path }}"]
+
+[tool.towncrier]
+directory = "news/"
+filename = "CHANGES.md"
+start_string = "<!-- towncrier release notes start -->\n"
+title_format = "## {version} ({project_date})"
+template = "news/.changelog_template.jinja"
+issue_format = "[#{issue}](https://github.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.python_package_name }}/issues/{issue})"
+underlines = ["", "", ""]
+
+[[tool.towncrier.type]]
+directory = "breaking"
+name = "Breaking changes:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "feature"
+name = "New features:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "bugfix"
+name = "Bug fixes:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "internal"
+name = "Internal:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "documentation"
+name = "Documentation:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "tests"
+name = "Tests"
+showcontent = true
+
+[tool.ruff]
+target-version = "py{{ cookiecutter.__python_version_identifier }}"
+line-length = 88
+fix = true
+lint.select = [
+    # flake8-2020
+    "YTT",
+    # flake8-bandit
+    "S",
+    # flake8-bugbear
+    "B",
+    # flake8-builtins
+    "A",
+    # flake8-comprehensions
+    "C4",
+    # flake8-debugger
+    "T10",
+    # flake8-simplify
+    "SIM",
+    # mccabe
+    "C90",
+    # pycodestyle
+    "E", "W",
+    # pyflakes
+    "F",
+    # pygrep-hooks
+    "PGH",
+    # pyupgrade
+    "UP",
+    # ruff
+    "RUF",
+]
+lint.ignore = [
+    # DoNotAssignLambda
+    "E731",
+]
+
+[tool.ruff.lint.isort]
+case-sensitive = false
+no-sections = true
+force-single-line = true
+from-first = true
+lines-after-imports = 2
+lines-between-types = 1
```

Comment:
I think we also need `order-by-type = false` 
Otherwise we get such diffs:

```diff
-from zope.interface import implementer
 from zope.interface import Interface
+from zope.interface import implementer
```

Reply:
Done